README.md

```diff
@@ -2,557 +2,87 @@
 
 ## Description
 ```
-```
-## Contibutors
-```
-Carrie Walsh(https://github.com/carriewalsh)
-Michael King-Stockton(https://github.com/KStockton)
-Ryan D Barnett(https://github.com/RyanDBarnett)
-Jennica Stiehl(https://github.com/stiehlrod)
-```
+Pocket Ramiro Rails is a backend service built in Rails that collaborates with two other services, Pocket Ramiro React[https://github.com/pocketramiro/pocket_ramiro_react] and Pocket Ramiro Documentation[https://master.d3bx8qvhmbmxu.amplifyapp.com/]. Together, theses three micro-services collaborate in order to provide facilities management for various small to medium sized businesses.
```

Comment:
Spelling error: Together, theses three ==> should be Together, these three

Reply:
Thanks